### DISCUSSION TITLE
E2E carbon responsibilities & other waste mapping

### DISCUSSION THREAD

**Discussion by mkorbi** [cncf/tag-env-sustainability#32] [Project Proposal]
Yet we are discussing a lot on how to measure which service is causing which consumption. There are a couple of ways how to do it from OpenTelemetry and Prometheus as well as old bare metal IPMI/RedFish implementations.

However, there are two (maybe more) issues:
- to know how much energy which component is using isn't telling us anything on the emissions it causes (important to know, but we are missing relevant data for it)
- we cover only Scope 2 [1], where as Scope 3 causes the heaviest pollution

I think we need to work here strategically with other organizations together (Mark from FinOps, to name one, has a strong drive that deserves/requires our support), and define our position, and help defining metrics and standards. We have to support initiatives to measure and report transparency of manufacturers as well as data center operators towards their carbon emissions.

Scope of this working group:
- Ensure seamless communication and collaboration
- Represent the cloud native community and deliver valuable input (e.g. domain knowledge, current trends and development)
- Help defining metrics or KPIs, data structures of taking in carbon emission data
- Design (or extend) an approach of splitting Scope 3 carbon emissions for digital services

Out of Scope:
- this group doesn't define, measure or develop solutions for gathering scope 3 emissions and their root cause

[1] https://www.nationalgrid.com/stories/energy-explained/what-are-scope-1-2-3-carbon-emissions

**Comment by leonardpahlke** [comment]
In summary: This working group should think about Scope 3 in our area and how we can work with others to find solutions and take action, + define KPIs, right?

I am -1 on this proposal at the moment, because it is not quite clear to me what the working group is supposed to do. What is the expected outcome, why should we start.

However, I do think that KPIs and collaboration with others in our field are something we need to work on.

**Reply by mkorbi** [reply]
No, not really. 

> Out of Scope:
> * this group doesn't define, measure or develop solutions for gathering scope 3 emissions and their root cause

This group should draw a reasonable line regarding scope 3. I believe we are utilizing that information, but scope 3 is nothing we directly can/will work on. However, as soon as this data hits "the cloud" we have to do something. 

To define this something should look like, which information we expect, what we can do with it, like assign it to services etc. that would be the idea for the task.

**Comment by rootfs** [comment]
While there are multiple complicated issues in classifying the scopes of emissions, some of them are out of the CNCF scope. We can still work towards certain best practices and solutions that will provide supporting evidence for discussions in that direction. 

In that line, let's focus on the exploration and document our achievement. The WG will be a perfect org to collect community input and amplify the findings.
